jcl/src/java.base/share/classes/jdk/internal/nicl/NativeInvoker.java

```diff
@@ -175,5 +183,73 @@ private static long getStructAddress(Reference<?> arg) throws IllegalAccessExcep
 		Pointer<?> ptr = RuntimeSupport.createPtr(addr, returnLayoutType);
 		return (Reference<?>) ptr.deref();
 	}
+	
+	private static long generateFunctionPointerBinding(Object anonInstance) {
+		Class<?> lambdaClass = anonInstance.getClass();
+		Method method = null;
+		Method[] arrayOfMethod;
+		int j = (arrayOfMethod = lambdaClass.getDeclaredMethods()).length;
+		for (int i = 0; i < j; i++) {
+			Method m = arrayOfMethod[i];
+			if (m.getName().contains("fn")) { //$NON-NLS-1$
+				method = m;
+			}
+		}
+		
+		long addr = generateNativeFunctionPointerBinding(method.getReturnType(), 
+														method.getParameterTypes(), 
```

Comment:
This is unreadable in the github review pane.  Please indent the arguments by on a few tabs so this more readable.

ie
```
+		long addr = generateNativeFunctionPointerBinding(method.getReturnType(), 
+						method.getParameterTypes(), 
```